reference/test/netflix_encryption_key.key

```diff
@@ -0,0 +1 @@
+eyJhbGciOiAiQTEyOEdDTSJ9.eyJjb250YWluZXJWZXJzaW9uIjogNSwgInJvb3RLZXlUeXBlIjogInVuaXF1ZSIsICJrZXlUeXBlIjogIkFFUy0xMjgiLCAiZW5jcnlwdGVkS2V5IjogIkVDZ0R1RUJjZVJERkFMYU43YmtzVFE9PSIsICJpdiI6ICI5R0RpdkR0M251SUU2OWhTIiwgImtleVVzYWdlIjogMywgImVudGl0bGVkVGFJZHMiOiBbIjE1N2Y3NjhmLWJhZDAtNDcwYi05MjlkLTBkN2RlYzI5ZDIyMCIsICIxNTdmNzY4Zi1iYWQwLTQ3MGItOTI5ZC0wZDdkZWMyOWQyMjEiLCAiMTU3Zjc2OGYtYmFkMC00NzBiLTkyOWQtMGQ3ZGVjMjlkMjIyIiwgIjE1N2Y3NjhmLWJhZDAtNDcwYi05MjlkLTBkN2RlYzI5ZDIyMyIsICIxNTdmNzY4Zi1iYWQwLTQ3MGItOTI5ZC0wZDdkZWMyOWQyMjQiLCAiMTU3Zjc2OGYtYmFkMC00NzBiLTkyOWQtMGQ3ZGVjMjlkMjI1IiwgIjE1N2Y3NjhmLWJhZDAtNDcwYi05MjlkLTBkN2RlYzI5ZDIyNiIsICIwMDAwMDAwMC0wMDAwLTAwMDAtMDAwMC0wMDAwMDAwMDAwMDEiXSwgImMxIjogInppT0RRaUg4ei9WSkZ5amNnKzFlYkE9PSIsICJjMiI6ICJLVWtWZHZUT20vdHV5K2V3NjBTRkVBPT0iLCAiYzMiOiAidmRjcjQ5NUJZM3VYVnZ0dGQ3eDBrQT09In0.Ziddaq2i37O4wflr_m4OiQ
```

Comment:
Please confirm that this is not sensitive information and is completely suitable for opensourcing.  Can the file name be changed to replect this?  (Something like "fake-netflix_encryption_key.key or "bogus_netflix..." etc.)